docs/UsersGuide/source/Graphics.rst

```diff
@@ -54,6 +54,8 @@ On Jet:
 
 On Orion: 
 
+NOTE DOM 20220227  - why is this not in a shared, user-independent directory? We have /work/noaa/nems/emc.nemspara and /work/noaa/nems/glopara, the latter already hosts the fixed files etc. A conditionn for tier-1 platforms was that no data/scripts/... resides in user directories. This disqualified stampede from being a tier-1 platform.
+
 .. code-block:: console
 
```

Comment:
We should move NaturalEarth on Orion to:

/work/noaa/global/glopara/fix/

I can send an email to Kate.

Reply:
I just moved the NaturalEarth files on Hera to a user-independent directory.  Line 47 can be changed to:

/scratch2/BMC/det/UFS_SRW_app/v1p0/fix_files/NaturalEarth

Reply:
I have updated to documentation to both of these paths so we need to make sure they are moved appropriately (talk to Kate about Orion).